test/unit/server/routes/health.spec.js

```diff
@@ -22,23 +28,33 @@ describe('GET /health', () => {
       }),
   );
 
-  context('when the build-info.json file is present', () => {
-    const projectRoot = path.resolve(__dirname, '../../../../');
-    const buildJson = path.resolve(projectRoot, 'build-info.json');
+  context('when the DB isn\'t working', () => {
```

Comment:
Noticed that you are not restoring stubs after the test suite. Is there any reason for that?.. Also usually I avoid stubs as it hides errors how come you went with this approach?